ui/src/pages/settings/repo-auto-imports/[importId].tsx

```diff
@@ -0,0 +1,139 @@
+import { useMutation, useQuery } from '@apollo/client'
+import { BreadcrumbNav, Button, Checkbox, ColoredBox, Panel, Toolbar } from '@mergestat/blocks'
+import type { NextPage } from 'next'
+import Head from 'next/head'
+import { useRouter } from 'next/router'
+import { Fragment, useEffect, useState } from 'react'
+import { SyncType } from 'src/@types'
+import { GetRepoImportQuery } from 'src/api-logic/graphql/generated/schema'
+import { UPDATE_AUTO_IMPORT_REPOS } from 'src/api-logic/graphql/mutations/add-repo'
+import { GET_REPO_IMPORT } from 'src/api-logic/graphql/queries/get-repo-imports'
+import Loading from 'src/components/Loading'
+import RepoImage from 'src/components/RepoImage'
+import { showSuccessAlert } from 'src/utils/alerts'
+import { SYNC_REPO_METHOD } from 'src/utils/constants'
+
+import SettingsView from 'src/views/settings'
+
+const AutoImportsDetail: NextPage = () => {
+  const [name, setName] = useState('')
+  const [syncsTypesArray, setSyncsTypesArray] = useState<SyncType[]>([])
+
+  const router = useRouter()
+  const { importId } = router.query
+
+  const { loading, data } = useQuery<GetRepoImportQuery>(GET_REPO_IMPORT, {
+    variables: { id: importId },
+    fetchPolicy: 'no-cache',
+  })
+
+  const [updateAutoImport] = useMutation(UPDATE_AUTO_IMPORT_REPOS, {
+    onCompleted: () => {
+      showSuccessAlert('Default syncs saved')
+      router.push('/settings/repo-auto-imports')
+    }
+  })
+
+  useEffect(() => {
+    const property = data?.repoImport?.type === SYNC_REPO_METHOD.GH_ORG ? 'org' : 'user'
+    setName(data?.repoImport?.settings[property])
+
+    const defaultSyncs = data?.repoImport?.settings.defaultSyncTypes
+
+    const list = data?.repoSyncTypes?.nodes.map(st => ({ type: st.type, description: st.description, shortName: st.shortName, checked: defaultSyncs.includes(st.type) }))
+    setSyncsTypesArray(list || [])
+
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
just curious about this - is there a reason to skip the lint step here?

Reply:
ohhh no for this case it is no needed. [aebfd74](https://github.com/mergestat/fuse/pull/377/commits/aebfd744874665fe3977e4a3eaffdc7e62402359)